README.md

```diff
@@ -20,6 +20,7 @@ Kickstart your Nix environments.
     - [Python (application)](#python-application)
     - [Python (package)](#python-package)
     - [Rust](#rust)
+    - [Node.js (backend)](#nodejs-backend)
```

Comment:
Don't hate me lol let's make sure this is in alphabetical order

Reply:
Will do (for both comments)